src/com/example/Demo_Subj/RoomView.java

```diff
@@ -22,17 +22,138 @@
+        return aktRoomID;
+    }
+
+    private void fillWalkingArrayLists(Resources resources){
+
+        List<Bitmap> subjectWalkingList;                                                                    //Animation des Subjekts und Laufens laden
+        Bitmap bm;                                                                                          //temporäre Bitmap Variable zum spiegeln und befüllen der Arrays
+
+        Bitmap subjectBStand = BitmapFactory.decodeResource(resources, R.drawable.subjekt);
+        subjectWalkingList = new ArrayList<Bitmap>();
+        subjectWalkingList.add(BitmapFactory.decodeResource(resources, R.drawable.walk_1));
+        subjectWalkingList.add(BitmapFactory.decodeResource(resources, R.drawable.walk_2));
+        subjectWalkingList.add(BitmapFactory.decodeResource(resources, R.drawable.walk_3));
+        subjectWalkingList.add(BitmapFactory.decodeResource(resources, R.drawable.walk_4));
+
+        subjStandBitmap = Bitmap.createScaledBitmap(subjectBStand, 170, 330, false);
+        subjStandBitmapInv = mirrorBitmap(subjStandBitmap);
+
+        aktBitmap = subjStandBitmap;
+
+        subjectWalk = new ArrayList<Bitmap>();                                                              //Arrays für die einzelnen Bilder der Animation (vorwärts und rückwärts)
+        subjectWalkInv = new ArrayList<Bitmap>();
+
+        for (int i = 0; i <= (subjectWalkingList.size() - 1); i++){                                         //Vorwärtsarray befüllen
+            bm = subjectWalkingList.get(i);
+            subjectWalk.add(i,  Bitmap.createScaledBitmap(bm, 170, 330, false));
+        }
+
+        for (int i = 0; i <= (subjectWalk.size() - 1); i++){                                                //Rückwärtsarray ist gespiegeltes Vorwärtsarray
+            bm = subjectWalk.get(i);
+            bm = this.mirrorBitmap(bm);
+            subjectWalkInv.add(i, bm);
+        }
+
+    }
+
+    private void setDefaultKoords(float xDef, float yDef, int room) {
+        xPos = xDef;
+        yPos = yDef;
+        aktRoomID = room;
+
+        xDest = xDef;
+        destRoomID = room;
+    }
+
+    private void fillRoomList(Resources resources){
+        roomList = new ArrayList<Room>();
+        World.setRoom(1, new Room(1, R.drawable.schlafzimmer, 0.0, 0.0, null, this.ctx));
+        World.setRoom(2, new Room(2, R.drawable.wohnzimmer, 0.0, 0.0, null, this.ctx));
+        //roomList.add(new Room(BitmapFactory.decodeResource(resources, R.drawable.schlafzimmer), 1, this.ctx));
+        //roomList.add(new Room(BitmapFactory.decodeResource(resources, R.drawable.wohnzimmer), 2, this.ctx));
+    }
+
+    private void fillObjectAnimList(Resources resources){
+        //eine globale Liste für Objektanimationen wird hier befüllt
+        //die Variable animation steht für eine Animation, die ausgeführt werden soll
+        //die switch-case Verzweigung in der Methode onDraw muss zusätzlich erweitert werden
```

Comment:
wenn du die anderen Änderungen gemacht hast, füge die Pflanze hier auch mit den Koordinaten zwischen 0 und 1 ein (also z.B. 0.5/0.5 - um in der Mitte des Bildes zu sein).